common/log/zap_logger.go

```diff
@@ -55,13 +56,13 @@ var _ Logger = (*zapLogger)(nil)
 
 // NewTestLogger returns a logger for tests
 func NewTestLogger() *zapLogger {
```

Comment:
There's a ticket to remove this function and replace it with testlogger.TestLogger.